composer.json

```diff
@@ -14,8 +14,8 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "symfony/console": "~2.3.10|~2.4.2|~2.5",
-        "nikic/php-parser": "~1.0",
+        "symfony/console": "~2.3.10|^2.4.2",
+        "nikic/php-parser": "^1.2.1",
```

Comment:
How long ago did the carat operator ship? I checked the composer changelog but didn't see it.